src/views/Pipeline.vue

```diff
@@ -502,6 +502,13 @@ export default defineComponent({
         event: ev,
         componentProps: { job }
       });
+
+      popover.onDidDismiss().then((result) => {
+        const jobEnumId = result.data?.systemJobEnumId;
+        if (jobEnumId && !this.getPinnedJobs.includes(jobEnumId)){
```

Comment:
Why this is needed?

Reply:
when a job is pinned and selected
Now if the user unpin that job, we have to updateSelectedPinnedJob without updating, the job is unpinned but that job is still filtered